src/clj/game/core/moving.clj

```diff
@@ -218,7 +224,7 @@
              (trigger-event state side :card-moved {:card card
                                                     :moved-card (get-card state moved-card)}))
            ;; move-zone-fn and the event can both modify the card, so re-bind here
-           (let [moved-card (get-card state moved-card)]
+           (when [moved-card (get-card state moved-card)]
```

Comment:
This is not right.

Reply:
oops, I made it when-let at some point (I tried a few different approaches here) and changed it back wrong :joy: